README.md

```diff
@@ -66,8 +63,8 @@ After cloning this repository, build with following command.
 mvn package 
 ```
 
-Running Mini Instance
----------------------
+Running via Maven
```

Comment:
I like the idea of making it not log when last was 0 and current is 0.  For now maybe example could be modified to pipe output to `grep -v "added 0 new notifications"`